### ISSUE TITLE
Bahasa Indonesia (id) translation 

### ISSUE THREAD

**Issue by kelimuttu** [mozilla/activate.mozilla.community#104]
Forked on https://github.com/kelimuttu/activate.mozilla.community 
will update here once the translation has been done.

**Comment by brianking** [comment]
We are currently overhauling our localisation system (issue #34) so please do not start anything until that is complete. Thanks for getting this started though. We'll let you know.

**Comment by kelimuttu** [comment]
Aah, ok. Let's put this on hold.

**Comment by MichaelKohler** [comment]
Sorry it took so long. After all this time, we now decided to go back to the old system so we don't block on any translations anymore. This means that the translations live in the GitHub repository here. You can get more information on how to localize in the README.

General flow:

* Fork this project
* Look at the following folders:
https://github.com/mozilla/activate.mozilla.community/tree/gh-pages/_pages/
https://github.com/mozilla/activate.mozilla.community/blob/gh-pages/_data/l10n/
https://github.com/mozilla/activate.mozilla.community/tree/gh-pages/_includes/
* If there is no subfolder for your locale, create one
* Copy the English files to the subfolder of your locale and start translating them, please note that strings can be in any of the above mentioned folders.
* Create a PR with your changes.
* We will then add the language to the language dropdown

Feel free to ping me for any questions you might have.

**Comment by MichaelKohler** [comment]
We are currently working on a refresh of the Activate page. This project is in its early stages, but what I can say now is that localization improvements (Pontoon) will be a big part of it. I'm closing this as it will be considered in the refresh. Once we will talk about about what locales to enable to be translated we will revisit all localization issues in this repository, so there is no need to create a new request somewhere.
